src/python/zquantum/core/wip/circuits/_circuit_test.py

```diff
@@ -55,6 +56,17 @@ def test_creating_circuit_has_correct_operations():
     assert circuit.operations == list(EXAMPLE_OPERATIONS)
 
 
+@pytest.mark.parametrize("circuit", [
+    Circuit(),
+    Circuit([]),
+    Circuit([H(0)]),
+    Circuit([H(0)], 5),
+])
+def test_printing_circuit_doesnt_raise(circuit):
```

Comment:
I would say name it `doesnt_raise_exception` , but I think it's a matter of taste ;)